### PR TITLE
APPLIED CLASSNAMES IN REACT VANILLA COMPONENTS

### PR DESCRIPTION
**Pull request by pavi41**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/react-vanilla-components/src/components/CheckBox.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const CheckBox = (props: PROPS) => {
   };
 
   return (
-    <div className="cmp-adaptiveform-checkbox" data-cmp-is="adaptiveFormCheckBox" data-cmp-visible={visible} data-cmp-enabled={enabled}>
+    <div className={appliedCssClassNames ? `cmp-adaptiveform-checkbox ${appliedCssClassNames}` : 'cmp-adaptiveform-checkbox'} data-cmp-is="adaptiveFormCheckBox" data-cmp-visible={visible} data-cmp-enabled={enabled}>
```
Comment (rismehta) [inline packages/react-vanilla-components/src/components/CheckBox.tsx:40]: Can you simplify this by using  className={`cmp-adaptiveform-checkbox ${appliedCssClassNames || ''}`}

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

**Review by amit-mnnit — CHANGES_REQUESTED** [review]
There are multiple conditions for adding a class. Please use the classnames package for this. 
 https://www.npmjs.com/package/classnames

File: packages/react-vanilla-components/src/components/CheckBox.tsx (hunk 1 above)
Comment (pavi41) [reply packages/react-vanilla-components/src/components/CheckBox.tsx:40]: Done!

**Pull request merged**